roles/cis/tasks/3/3.1.1.yml

```diff
@@ -22,7 +22,7 @@
         reload: no
         ignoreerrors: yes
       delegate_to: localhost
-    - name: "set net.ipv6.conf.all.forwarding"
+    - name: "3.1.1 set net.ipv6.conf.all.forwarding to 0"
       sysctl:
         name: net.ipv6.conf.all.forwarding
         value: 0
```

Comment:
quote all these values please. value: '0'

Reply:
I will check this and update you.

Reply:
Fixed!